cmd/skaffold/app/cmd/flags.go

```diff
@@ -573,6 +573,14 @@ var flagRegistry = []Flag{
 		},
 		IsEnum: true,
 	},
+	{
+		Name:          "id",
+		Usage:         "Survey id for survey command to open.",
+		Value:         &surveyID,
+		DefValue:      constants.HaTS,
+		FlagAddMethod: "StringVar",
+		DefinedOn:     []string{"survey"},
+	},
```

Comment:
Given that this argument isn't used on other commands, we can just move this into `NewCmdSurvey()`.

Reply:
done